spec/elastic_apm/spies/mongo_spec.rb

```diff
@@ -89,28 +92,25 @@ def start_mongodb
       expect(db.type).to eq 'mongodb'
       # ParallelCollectionScan doesn't send 'lsid' in the command so we can
       # validate the entire command document.
-      expect(db.statement).to eq '{"parallelCollectionScan"=>"testing", ' \
-        '"numCursors"=>1}'
+      expect(db.statement).to match('"delete"=>"testing"')
```

Comment:
You can remove the comment about ParallelCollectionScan but I still don't understand how you can validate the entire delete command? When I was writing this test, the `lsid` value was in the command so I couldn't use delete and chose ParallelCollectionScan instead. Is there some setup somewhere I'm not aware of?

Reply:
For example, I get this `"{\"delete\"=>\"testing\", \"ordered\"=>true, \"deletes\"=>[{\"q\"=>{}, \"limit\"=>0}], \"lsid\"=>{\"id\"=><BSON::Binary:0x70210864279520 type=uuid data=0x4c4b83c656b94211...>}}"`

Reply:
`match(String)` (as opposed to `eq()`) uses `include?` under the hood, I think, so it's just a matter of the string being in there somewhere.

The reason I changed it at all was that my Mongo (that I finally figured out had moved to their own homebrew tap) was telling me that `parallelCollectionScan` had been removed.

Reply:
Oh great, I didn't notice you used `match` instead. This works too!